src/common/stats/StatsManager.h

```diff
@@ -65,7 +65,9 @@ class StatsManager final {
     static void setReportInfo(HostAddr addr, int32_t interval);
 
     // Both register methods return the index to the internal data structure.
-    // This index will be used by addValue() methods
+    // This index will be used by addValue() methods.
+    // Both register methods is not thread safe, and user need to register stats one
+    // by onebefore calling addValue, readStats, readHisto and so on.
     static int32_t registerStats(folly::StringPiece counterName);
```

Comment:
You better make it to be private,  and expose a method named "init"  to do "register stats"

Reply:
Because the stats of graph/meta/storage is not the same, and we have to specify a counter is a stat or histogram type, so maybe we'd better register them in each module. 

@laura-ding, WDYT?

Reply:
![image](https://user-images.githubusercontent.com/48548375/69402586-3d3b2180-0d33-11ea-90c8-c745cb995fc0.png)
This is graphd/metad/storaged that can be used together, you can put it in StatsManager as a common function. @critical27

Reply:
> ![image](https://user-images.githubusercontent.com/48548375/69402586-3d3b2180-0d33-11ea-90c8-c745cb995fc0.png)
> This is graphd/metad/storaged that can be used together, you can put it in StatsManager as a common function. @critical27

Good suggestion, we could do it in #1301